### DISCUSSION TITLE
How to handle IPv6 traffic that connect use IP when only use TUN inbound

### DISCUSSION THREAD

**Discussion by EkkoG** [eycorsican/leaf#130] [General]
For now, traffic that uses IPv6 IP directly not handle by TUN, Chinese international IPv6 bandwidth is so small, about 100Gbps totally, connect use IPv6 directly is not very good, and proxy servers mostly do not have IPv6 outbound or not config. So it seems to let all IPv6 traffic go out directly is ok, but it's worth discussing how to handle IPv6 traffic if all requirements are met.

**Comment by eycorsican** [comment]
Have you configure an IPv6 address on the TUN interface and also add the corresponding IPv6 routes?
